src/Resources/app/administration/src/service/adyenService.js

```diff
@@ -69,6 +69,33 @@ class ApiClient extends ApiService {
                 throw error;
             });
     }
+
+    fetchNotifications(orderId) {
+        const headers = this.getBasicHeaders({});
+
+        return this.httpClient
+            .get(this.getApiBasePath() + '/orders/' + orderId + '/notifications', {
+                headers
+            })
+            .then((response) => {
+                return ApiService.handleResponse(response);
+            }).catch((error) => {
+                console.error('An error occurred: ' + error.message);
+                throw error;
+            });
+    }
+
+    isAdyenOrder(order) {
```

Comment:
@Morerice I copied this function into the `adyenService` so that I could reuse it in the `adyen-notifications` component